hack/templates/resources/30-cloudprofile.yaml.tpl

```diff
@@ -431,6 +433,101 @@ spec:<% caBundle=value("spec.caBundle", "") %>
         - cn-beijing-f  # Avalibility zone
       % endif
   % endif
+  % if cloud == "packet":
+  packet:
+    constraints:
+      dnsProviders:<% dnsProviders=value("spec.packet.constraints.dnsProviders", []) %>
+      % if dnsProviders != []:
+      ${yaml.dump(dnsProviders, width=10000)}
+      % else:
+      - name: aws-route53
+      - name: unmanaged
+      % endif
+      kubernetes:<% kubernetesVersions=value("spec.packet.constraints.kubernetes.versions", []) %>
+        % if kubernetesVersions != []:
+        ${yaml.dump(kubernetesVersions, width=10000)}
+        % else:
+        versions:
+        - 1.13.3
+        % endif
+      machineImages:<% machineImages=value("spec.packet.constraints.machineImages", []) %>
+      % if machineImages != []:
+      ${yaml.dump(machineImages, width=10000)}
+      % else:
+      - name: "Container Linux - Stable"
+        id: d61c3912-8422-4daf-835e-854efa0062e4
+      % endif
+      machineTypes:<% machineTypes=value("spec.packet.constraints.machineTypes", []) %>
+      % if machineTypes != []:
+      ${yaml.dump(machineTypes, width=10000)}
+      % else:
+      - name: t1.small
+        cpu: "4"
+        gpu: "0"
+        memory: Gi
+        usable: true
+        zones:
```

Comment:
For packet is it also the case that certain machine type are only available in certain availability zones? That's why you need to maintain this list of zones here?

Reply:
Well, yes and no. It just takes time for certain types to make it out to certain zones. I originally put it in, but it was complicating things too much. For now, I am removing it and having all types in all zones. We can add greater nuance to this later. Will remove here too.